packages/cfd/build/constants.js

```diff
@@ -1,6 +1,6 @@
 const CircularDependencyPlugin = require('circular-dependency-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
-const CopyPlugin = require('copy-webpack-plugin');
+// const CopyPlugin = require('copy-webpack-plugin');
```

Comment:
@Bahar-FS is this change expected in this PR? 🤔

Reply:
yes @farzin-deriv, after the revert of trading hub, these caused tests fail